utils/openocd/Makefile

```diff
@@ -8,14 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=openocd
-PKG_SOURCE_VERSION:=v0.11.0-rc1
+PKG_SOURCE_VERSION:=v0.11.0
 PKG_VERSION:=$(PKG_SOURCE_VERSION)
```

Comment:
Not needed. PKG_VERSION gets set from PKG_SOURCE_VERSION

Reply:
Noted